### PR TITLE
feat: IpfsImage component

### PR DESCRIPTION
**Pull request by chrispanag**

### CODE REVIEW

File: src/components/IpfsImage.vue [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<template>
+	<div v-if="imageSrc !== null" class="modal-animation">
+		<img v-lazy="imageSrc" :class="imgClass + ' object-cover'" />
+	</div>
+	<div v-else class="animate-pulse bg-gray1 dark:bg-gray7"></div>
```
Comment (irshadshalu) [inline src/components/IpfsImage.vue:5]: Nice! I can see this being re-usable in a lot of situations!

File: src/components/IpfsImage.vue [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	imageSrc.value = props.defaultImage;
+	if (props.cid) {
+		getPhotoFromIPFS(props.cid).then((photo) => {
+			imageSrc.value = photo;
+		});
+	}
+});
+</script>
+
+<!-- Right side: Image -->
```
Comment (irshadshalu) [inline src/components/IpfsImage.vue:30]: to be removed?

**Review by irshadshalu — APPROVED** [review]
lg2m

File: src/components/IpfsImage.vue (hunk 2 above)
Comment (chrispanag) [reply src/components/IpfsImage.vue]: yeap

**Review by irshadshalu — APPROVED** [review]

**Pull request merged**